### ISSUE TITLE
Wrap fix for `FiniteDifferences` into a Package Extension

### ISSUE THREAD

**Issue by mofeing** [bsc-quantic/Tensors.jl#11]
When using `ChainRulesTestUtils` to test our `frules` and `rrules`, `FiniteDifferences` crashes because the serialization procedure `FiniteDifferences.to_vec` doesn't work well with `Tensor{T,0}`.

The reason is that `to_vec` tries to serialize the `labels` field which in the case of `Tensor{T,0}` is an empty tuple, `Tuple{}`.

Right now, the testing code has a small fix that would be nice to export as a Package Extension when Julia 1.9 arrives.

```julia
# fix for FiniteDifferences: `FiniteDifferences.to_vec` fails for Tensor{T,0}
function ChainRulesTestUtils.FiniteDifferences.to_vec(x::T) where {T<:Tensor}
    x_vec, Array_from_vec = ChainRulesTestUtils.FiniteDifferences.to_vec(parent(x))
    _labels = labels(x)
    _meta = x.meta

    Tensor_from_vec(y) = T(Array_from_vec(y), _labels; _meta)

    return x_vec, Tensor_from_vec
end
```

**Comment by mofeing** [comment]
Fixed in a212c9fcb5a0dba5ff3d4bb908fac34bc6c41107

**Comment by mofeing** [comment]
Triggering TagBot for merged registry pull request: https://github.com/bsc-quantic/Registry/pull/38
